siliconcompiler/report/streamlit_viewer.py

```diff
@@ -632,10 +635,15 @@ def show_title_and_runs(title_col_width=0.7):
     metrics_tab, manifest_tab, file_preview_tab = tabs
 
 with metrics_tab:
+    if ui_width <= 0:
+        # in case streamlit_javascript errors
+        ui_width = 1
     if streamlit.session_state['flowgraph']:
-        node_from_flowgraph, datafram_and_node_info_col = show_flowgraph()
+        node_from_flowgraph, datafram_and_node_info_col = \
+            show_flowgraph(flowgraph_col_width=min(520 / ui_width, 0.4))
     else:
-        node_from_flowgraph, datafram_and_node_info_col = dont_show_flowgraph()
+        node_from_flowgraph, datafram_and_node_info_col = \
+            dont_show_flowgraph(flowgraph_col_width=min(120 / ui_width, 0.1))
```

Comment:
what do 520 and 120 mean? maybe you can make those variables so they get human readable names?